### PR TITLE
Add pyproject.toml for settings

### PR DESCRIPTION
**Pull request by kotamanegi**
This PR adds pyproject.toml for storing settings.
pyproject.toml contains:
- flake8 config
- `requirements.txt` dependencies information
- `setup.py`

self-introduction: I'm maintaining Qulacs at Osaka University. QuantestPy seems like a good realistic idea (and I want to contribute to this project!)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["wheel", "setuptools", "setuptools_scm"]
```
Comment (junnaka51) [inline pyproject.toml:2]: @kotamanegi 
Hi, thank you so much for having interests in this project! 😄 
I am new to pyproject.toml and wondering if `"setuptools_scm"` is necessary?

Comment (kotamanegi) [reply pyproject.toml:2]: `setuptools_scm` is not so much necessary (in this project).
I've removed "setuptools_scm".

Comment (kotamanegi) [inline pyproject.toml:2]:
```suggestion
requires = ["wheel", "setuptools"]
```

Comment (junnaka51) [reply pyproject.toml]: Thanks!

**Review by junnaka51 — APPROVED** [review]
LGTM 🙆 ( cc @mitz1012 )

**Review by junnaka51 — APPROVED** [review]

**Comment by junnaka51** [comment]
@mitz1012 @kotamanegi 
Github actions does not start... 😕 Any suggestions?

**Comment by kotamanegi** [comment]
I believe @mitz1012 have a right to give permission of running GitHub Actions for first-time contributer (like @junnaka51 and @kotamanegi).
This was introduced in [2021/04](https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs/) because actions were being hacked to Bitcoin mining.

**Review by mitz1012 — APPROVED** [review]
Thanks!

**Comment by mitz1012** [comment]
@kotamanegi 
申し訳ございませんがCIが動かないため、もう一度pull requestを作り直していただくことは可能でしょうか。
その際に、conflictが発生すると思うので併せてご対応いただけますと幸いです。

**Pull request closed without merging**